assets/src/edit-story/components/library/panes/media/dropDownMenu.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { useState, useRef } from 'react';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import DropDownList from '../../../../components/form/dropDown/list';
+import Popup from '../../../../components/popup';
+import { ReactComponent as More } from '../../../../icons/more_horiz.svg';
+
+const MoreIcon = styled(More)`
```

Comment:
This needs to be a proper `button` for accessbility reasons.

I recommend defining it in `assets/src/edit-story/components/button/index.js` together with other buttons:

```js
import { ReactComponent as MoreIcon } from '../../../../icons/more_horiz.svg';

// ...

export const More = (props) => (
  <StyledButton {...props}>
    <MoreIcon />
  </StyledButton>
);
```

Reply:
Done.